### ISSUE TITLE
Issues Mini-Internet on Debian 12.5

### ISSUE THREAD

**Issue by mark-hgb** [nsg-ethz/mini_internet_project#41]
Dear all,

I am working with a test system for the current Mini-Internet implementation based on a Debian 12.5 system. I am using the config from the 2022 assignment of the communication network course for testing purposes. I have two issues with this setting:

1. When starting the Mini-Internet with startup.sh I get "RTNETLINK answers: File exists" errors in host_links_setup.sh, which I am not able to locate.
2. In layer2_setup.sh I get an error in line 81 "row_l[6]: unbound variable". It seems that layer2_setup.sh is expecting a 7th row (buffer) in the l2_links.txt config which is not present or documented somewhere.

Can you please help me with these problems? Thanks for the wonderful software, I am using an older version of the Mini-Internet in my lectures and it works like a charm!

Best regards
Markus Zeilinger

**Comment by NotSpecial** [comment]
Hello!

Could you let me know which branch of the routing project you currently use for your tests?

We have just completed this year's iteration of the project for our lecture with a lot of improvements to the infrastructure and setup. The config has changed a bit, but maybe this will still be interesting to you. Currently, you can find it in the `commnet24` branch, but we'll soon merge the updates back into the main branch.

Best,
Alex

**Comment by mark-hgb** [comment]
Hi Alex,
I was using the master branch, but also tested the commnet24 branch now too. This one seems to work properly. Is it possible that the link throughput and delay stuff is not working anymore in the master branch as documented in the wiki? Maybe it is expecting some different kind of syntax, e. g. 10mbit for throughput or 10ms for delays.
Thanks again for this great piece of software. I am using the Mini-Internet for the third time this year in a lecture on advanced networking on master level and it is really, really helpful. 
Markus

**Comment by NotSpecial** [comment]
Yes, the wiki may not add up 100% currently, I'm afraid, since we merged a PR that added some link delays.

There were a few delays on our side, but we are not working on merging the commnet24 branch back into main, and along with that we'll also make a pass over the wiki to ensure things are lining up again.
